packages/acs-ui-javascript-loaders/src/callCompositeLoader.ts

```diff
@@ -22,41 +22,41 @@ import { initializeIcons } from '@fluentui/react';
 
 /**
  * Props for the OutboundCallComposite that you can use in your application.
```

Comment:
Update this comment for each composite type, its generic and doesn't say anything the name already tells the developer. Here is a good place to explain why there are two options bag and `@link` to where they can read more about each